testnet/litentry.json

```diff
@@ -0,0 +1,92 @@
+{
```

Comment:
It's rather better to have a script to generate this. I am not sure if this is possible. But the problem is if the runtime got updated, this may no longer be valid

Reply:
Seems that this can be produced in the testnet.md, then it's better not to include this file